docs/docs/09-tools.md

```diff
@@ -39,6 +39,13 @@ https://azure.microsoft.com/services/storage/blobs
 
 Azure Blob Storage เป็นพื้นที่จัดเก็บไฟล์บนคลาวด์ของทาง Microsoft Azure ที่เหมาะสำหรับ Unstructured data โดยเฉพาะ
 
+## Dash
+
+https://dash.plotly.com/  
+https://github.com/plotly/dash
```

Comment:
I think we could make the header a hyperlink

Reply:
like this?

```
## [Dash](https://dash.plotly.com/)
```

Reply:
✅